src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb.model.ts

```diff
@@ -26,3 +26,29 @@ export interface AuthMode {
   user: 'User';
   activeDirectory: 'active-directory';
 }
+
+export interface SMBShare {
+  cluster_id: string;
+  share_id: string;
+  intent: string;
+  name?: string;
+  readonly?: boolean;
+  browseable?: boolean;
```

Comment:
nit: move optional props down

Reply:
It was initially in the same order as in the docs, I've updated it to have optional fields last